spartan/scripts/test_spartan.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# NOTE! currently does not work with 'cannot validate constraint' on tx simulate via pxe
+set -eu
+set -o pipefail
+
+# Test a deployed cluster
+
+NAMESPACE=${1:-spartan}
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+if [ -z "$NAMESPACE" ]; then
+  echo "Usage: $0 (optional: <namespace>)"
+  echo "Example: $0 devnet"
+  exit 1
+fi
+
+echo "Note: Repo should be bootstrapped with ./bootstrap.sh fast."
+
+# Fetch the service URLs based on the namespace
+export BOOTNODE_URL=http://$(kubectl get svc -n $NAMESPACE -o jsonpath="{.items[?(@.metadata.name=='$NAMESPACE-aztec-network-boot-node-lb-tcp')].status.loadBalancer.ingress[0].hostname}"):8080
```

Comment:
function?

Reply:
get the load balancer URLs to inject into the script

Reply:
sorry I meant a function could be used for dedupe